jquery/footHovzer.d.ts

```diff
@@ -0,0 +1,32 @@
+declare global {
+    interface JQueryStatic {
+        /**
+         * Utility to stack stuff in an overlay fixed on the bottom of the page.
+         *
+         * Usage:
+         *
+         * ```js
+         * var hovzer = $.getFootHovzer();
+         * hovzer.$.append( $myCollection );
+         * hovzer.update();
+         * ```
+         *
+         * @returns {FootHovzer}
+         */
+        getFootHovzer(): FootHovzer;
```

Comment:
Which RL module is this part of? 

I am getting `$.getFootHovzer is not a function` even after running `mw.loader.using(Object.keys(mw.loader.moduleRegistry).filter(m => m.startsWith('jquery.')))` on mediawiki.org.

Reply:
It is part of `mediawiki.debug`, you can find it in [`resources/src/mediawiki.debug/jquery.footHovzer.js`](https://github.com/wikimedia/mediawiki/blob/REL1_41/resources/src/mediawiki.debug/jquery.footHovzer.js)

Reply:
Oh, I'm sorry, I missed the `@private` annotation on the module comment. Private members seem to currently mostly being included, but this is more or less an entire module, so I don't know whether it should be included. For now I've just added the annotation.